CMakeLists.txt

```diff
@@ -277,10 +294,16 @@ endif()
 # Set override properties
 # -----------------------------------------------------------------------------
 if (MI_OVERRIDE MATCHES "ON")
-  target_compile_definitions(mimalloc PRIVATE MI_MALLOC_OVERRIDE)
+  if (MI_BUILD_SHARED)
```

Comment:
It is not clear why this is only in `MI_BUILD_SHARED` -- the override should be defined in all targets as all can be used to override standard malloc definitions.

Reply:
It is there, see below. mimalloc target is a shared library, so it is available only when MI_BUILD_SHARED is on. Probably it would make sense to have mimalloc-shared target to remove the confusion?

Reply:
Ah I see